tools/contrib/ssp_anomaly_forcing_smooth

```diff
@@ -196,25 +199,15 @@ if num_ens != len(histdir):
     print("number of ensemble members not the same")
     sys.exit("number of members different")
 
-# test w/ 1 ensemble member
-num_ens = 3
-
 # Setup output directory
-sspoutdir = "anomaly_forcing/CMIP6-" + ssptag
+sspoutdir = "CMIP6-" + ssptag
```

Comment:
fwiw I like this anomaly_forcing subdirectory that was created previously

Reply:
(See [above](https://github.com/ESCOMP/CTSM/pull/2181/files#r1380747997).)